docroot/themes/custom/uids_base/templates/uids/button.html.twig

```diff
@@ -0,0 +1,10 @@
+{% set attributes = attributes ?: create_attribute() %}
+<div {{ attributes }}>
+  {% block button_content_start %}
+    {{ title_prefix }}
+    <a {{ button_classes }} href="{{ button_link_url }}">
+      {{ button_link_title }}
```

Comment:
I'd be interested in moving away from having `button_`-prefixed variables names, at least in the context of this file.

Reply:
I will try and refactor!

Reply:
This hsa been achieved!